### ISSUE TITLE
Download an application's attachments

### ISSUE THREAD

**Issue by elliottSlee** [bcgov/CONN-CCBC-portal#757]

**Comment by AntBush** [comment]
This, assuming we're allowing people to upload potentially infected files (while the anti-virus check is not done), needs some states and AC for how we want to handle alerting that a file they're downloading is infected (or whatever may be the case after we figure out what we want to do with infected files)

**Comment by elliottSlee** [comment]
@AntBush Does that mean this is blocked by #275?

**Comment by elliottSlee** [comment]
Should we wireframe some alerts or will #275 remove the need for them?

**Comment by matthieu-foucault** [comment]
@elliottSlee I have updated the acceptance criteria to add the error state when the file is quarantined. We should work out what a helpful user message is.

For analysts, we can be upfront about the reason why the download fails.

For applicants (if this is in scope for this issue), it should be a generic error message

**Comment by eazeredo** [comment]
@matthieu-foucault , why not return a custom error for the Analyst (internal user, not for the applicant) informing that the file is in quarantine? Is there any security reason for that?

**Comment by matthieu-foucault** [comment]
> why not return a custom error for the Analyst

This is what I am suggesting in my comment above

> For analysts, we can be upfront about the reason why the download fails.

**Comment by elliottSlee** [comment]
In our discussion on Monday, I thought we landed on not doing that. cc @wenzowski

**Comment by matthieu-foucault** [comment]
Also yes, given the low probability of infected files, implementing a custom error is a low priority (i.e. shouldn't be in out backlog)

**Comment by MarsRomer** [comment]
I'm considering deprioritizing this to post MVP - for now the sharepoint site may have to do for viewing files
